src/Microsoft.AspNetCore.Mvc.ViewFeatures/AutoValidateAntiforgeryTokenAttribute.cs

```diff
@@ -22,7 +22,11 @@ namespace Microsoft.AspNetCore.Mvc
     public class AutoValidateAntiforgeryTokenAttribute : Attribute, IFilterFactory, IOrderedFilter
     {
         /// <inheritdoc />
```

Comment:
Inheritdoc doesn't really work with these kinds of thing. you'll want to also provide a summary here